test/gpu_tests.jl

```diff
@@ -121,6 +121,28 @@ end
     end
 end
 
+@kernel function test_1_moment_micro_smooth_transition_kernel!(
+    acnv1M,
+    output::AbstractArray{FT},
+    ql
+) where {FT}
+
+    i = @index(Group, Linear)
```

Comment:
I think we need to use `Global` here? xref: https://github.com/CliMA/SurfaceFluxes.jl/issues/142

Reply:
Thank you! I wonder how the tests are passing with that. 

I'll fix it following what you did in SurfaceFluxes